db_master_deploy/dml_trigger.sh

```diff
@@ -85,22 +85,26 @@ do
     echo "opening ssh connection to ${target} sphinx host: ${sphinx} ..."
     ssh -T ${sphinx} /bin/bash << HERE
             if [ -f ${SPHINX_CONFIG} ]; then
-                for table in ${tables}
-                do
-                    echo "  update sphinx indexes that use the database source: \${table} ..."
-                    python -u ${SPHINX_PG_TRIGGER} -d \${table} -c update -s ${SPHINX_CONFIG}
-                done
-                sleep 2
-                # check service status, if not running start service
-                if  ! searchd --status  > /dev/null
+                # silent check of service status, if searchd is not responding stop and start systemctl and searchd
+                if  ! searchd --status  &> /dev/null
                 then
                     echo "sphinx service was not running, trying to start sphinx service on host ${sphinx}"
-                    sudo -u root systemctl stop sphinxsearch
+                    sudo -u sphinxsearch searchd --stop &> /dev/null || :
```

Comment:
why to methods to stop it

Reply:
thats exactly the point, i cannot reproduce or tell you why searchd can still run while systemctl thinks the service is not running anymore.